Azure/sonic-buildimage
# [Mellanox] Add support for a new platform LS-SN2700

Signed-off-by: Andriy Moroz <c_andriym@mellanox.com>

**- What I did**
Added config files for the new Mellanox platform LS-SN2700

**- How I did it**
Added configs under devices/mellanox/

**- How to verify it**
Build image, install on device platform=x86_64-mlnx_lssn2700-r0, HWSKU=LS-SN2700 and make sure it is configured properly

**- Description for the changelog**
Added new platform support Mellanox LS-SN2700
